src/game/logic/object/weapon.cpp

```diff
@@ -1019,10 +1019,10 @@ float Weapon::Get_Percent_Ready_To_Fire() const
 
             if (now < next_shot) {
                 captainslog_dbgassert(next_shot >= m_whenLastReloadStarted, "next_shot >= m_whenLastReloadStarted");
-                unsigned int time_left = next_shot - m_whenLastReloadStarted;
```

Comment:
Should we also replace m_whenLastReloadStarted with the getter Get_Last_Reload_Started()?

Reply:
original code didn't and seems they followed a rule to avoid getters inside classes that define the member